doniclient/tests/osc/test_hardware.py

```diff
@@ -1,31 +1,37 @@
 from doniclient.osc import cli as hardware_cli
 from doniclient.tests.osc import fakes as hardware_fakes
-
+from osc_lib import utils as oscutils
+from osc_lib import exceptions
+import unittest
 FAKE_HARDWARE_UUID = hardware_fakes.hardware_uuid
+FAKE_HARDWARE_NAME = hardware_fakes.hardware_name
 
 UPDATE_PARAMS = [
     (
         "baremetal",
-        "--mgmt_addr",
-        "mgmt_addr",
-        "/properties/mgmt_addr",
+        "--management_address",
+        "management_address",
+        "/properties/management_address",
         "fake-mgmt_addr",
+        True,
```

Comment:
Same as other comment, I'm assuming that a rebase will remove from this patch-set.